nltk/parse/stanford.py

```diff
@@ -473,16 +473,6 @@ def _make_tree(self, result):
         return DependencyGraph(result, top_relation_label="ROOT")
 
 
-@skip("doctests from nltk.parse.stanford are skipped because it's deprecated")
+@pytest.mark.skip(reason="doctests from nltk.parse.stanford are skipped because it's deprecated")
 def setup_module(module):
-    from nose import SkipTest
```

Comment:
this looks like unreachable code, right? since we marked the whole thing as skip